client/web/src/enterprise/site-admin/dotcom/productSubscriptions/LlmProxyRateLimitModal.tsx

```diff
@@ -54,8 +62,12 @@ export const LLMProxyRateLimitModal: React.FunctionComponent<React.PropsWithChil
                     variables: {
                         productSubscriptionID,
                         llmProxyAccess: {
-                            rateLimit: limit,
-                            rateLimitIntervalSeconds: limitInterval,
+                            chatCompletionsRateLimit: mode === 'chat' ? limit : undefined,
+                            chatCompletionsRateLimitIntervalSeconds: mode === 'chat' ? limitInterval : undefined,
+                            chatCompletionsAllowedModels: mode === 'chat' ? splitModels(allowedModels) : undefined,
+                            codeCompletionsRateLimit: mode === 'code' ? limit : undefined,
+                            codeCompletionsRateLimitIntervalSeconds: mode === 'code' ? limitInterval : undefined,
+                            codeCompletionsAllowedModels: mode === 'code' ? splitModels(allowedModels) : undefined,
```

Comment:
maybe add a line here for readability?

```suggestion
                            chatCompletionsRateLimit: mode === 'chat' ? limit : undefined,
                            chatCompletionsRateLimitIntervalSeconds: mode === 'chat' ? limitInterval : undefined,
                            chatCompletionsAllowedModels: mode === 'chat' ? splitModels(allowedModels) : undefined,

                            codeCompletionsRateLimit: mode === 'code' ? limit : undefined,
                            codeCompletionsRateLimitIntervalSeconds: mode === 'code' ? limitInterval : undefined,
                            codeCompletionsAllowedModels: mode === 'code' ? splitModels(allowedModels) : undefined,
```